data/src/test/java/org/oppia/android/data/backends/gae/NetworkLoggingInterceptorTest.kt

```diff
@@ -52,9 +54,12 @@ class NetworkLoggingInterceptorTest {
     private const val headerString = "$testApiKey: $testApiKeyValue"
   }
 
-  @Inject lateinit var networkLoggingInterceptor: NetworkLoggingInterceptor
-  @Inject lateinit var context: Context
-  @Inject lateinit var testCoroutineDispatchers: TestCoroutineDispatchers
+  @Inject
+  lateinit var networkLoggingInterceptor: NetworkLoggingInterceptor
+  @Inject
+  lateinit var context: Context
+  @Inject
+  lateinit var testCoroutineDispatchers: TestCoroutineDispatchers
```

Comment:
I prefer the format because it's less lines & easier to read. Eventually we'll be moving over to ktfmt which will hard enforce an exact style #2799 since ktlint is a bit too permissive in my opinion (I'd rather try and make all code look as though it was written by the same author).

There's not a lot we can do about Android Studio's formatting, unfortunately. I haven't found great ways of configuring it to actually match ktlint or ktfmt, but perhaps there are plugins that can help with his (I haven't looked in a long time).

FWIW from a reviewer side, I tend to only ask people to change it if the existing code is already single line (for consistency).